graph/max_flow_with_ford_fulkerson_and_edmond_karp_algo.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * Author: Amit Kumar
+ * Created: May 24, 2020
+ * Copyright: 2020, Open-Source
+ * 
+ * As cpplint checker doesn't allow me to use more then 80 columns 
+ * so i used some naming convention , which can be expended as: 
+ *
+ * r_capacity   -> residual_capacity 
+ * capacity     -> original edge capacity 
+ * vis          -> visited node 
+ * c_n          -> current_node
+ * n            -> total number of nodes
+ * m            -> total number of edges
+ * s            -> source 
+ * t            -> sink 
```

Comment:
Please use real variable names and wrap lines like the rest of the world does.

Reply:
sure